src/actions/users.js

```diff
@@ -230,12 +230,18 @@ export function loadMe() {
 }
 
 export function logout() {
-    return bindClientFunc(
-        Client4.logout,
-        UserTypes.LOGOUT_REQUEST,
-        UserTypes.LOGOUT_SUCCESS,
-        UserTypes.LOGOUT_FAILURE,
-    );
+    return async (dispatch, getState) => {
+        dispatch({type: UserTypes.LOGOUT_REQUEST}, getState);
+
+        try {
+            Client4.logout();
```

Comment:
This needs an await since it's asynchronous

Reply:
Actually I don’t think we need to wait for it to complete, we really don’t care do we?